.github/workflows/avocado.yml

```diff
@@ -0,0 +1,38 @@
+name: "[TEST-IGNORE] Swagger Avocado"
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  avocado:
+    name: "[TEST-IGNORE] Swagger Avocado"
+
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Must include all branches, for git branch logic in Avocado to work correctly
+          fetch-depth: 0
+
+      - name: Setup Node and install deps
+        uses: ./.github/actions/setup-node-install-deps
+
+      - name: Run Avocado
+        run: |
+          npm exec --no avocado \
+          --excludePaths \
+            "/common-types/" \
+            "/scenarios/" \
+            "/package.json" \
+            "/package-lock.json" \
+            "/cadl/examples/" \
```

Comment:
Do we still have cadl paths?

Reply:
We do.  Random example:

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/voiceservices/resource-manager/Microsoft.VoiceServices/cadl/examples/Contacts_Get.json